lib/public/views/Logs/Create/LogCreationModel.js

```diff
@@ -125,6 +130,16 @@ export class LogCreationModel extends Observable {
             options.headers['Content-Type'] = 'application/json';
         }
 
+        // The field detectorOrSubsystem and issueDescription should be allowed only if the log templateKey is On-call
+        if (templateKey) {
+            options.body.templateKey = templateKey;
+
+            if (templateKey === 'on-call') {
+                options.body.issueDescription = issueDescription;
+                options.body.detectorOrSubsystem = detectorOrSubsystem;
```

Comment:
Why do you want to send these info to the backend?

Reply:
done